recipes/nifti_clib/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+make CFLAGS="-I$PREFIX/include -L${PREFIX}/lib" LDFLAGS=-L${PREFIX}/lib CC=${CC} all
+mv include/* ${PREFIX}/include/
```

Comment:
I think it is better if you use `cp` instead of `mv`, e.g.: `cp -r include $PREFIX`